### DISCUSSION TITLE
Oh my Posh terminal setting doesn't reflect to remote ssh terminal or with docker container termial

### DISCUSSION THREAD

**Discussion by ammaryasirnaich** [JanDeDobbeleer/oh-my-posh#4707] [General]
I have "Oh my posh" setup locally which works well with the local terminal and with vscode, however via remote vscode connection to ssh server or docker the terminal, it doesn't reflect the settings. I don't know if there is any other setting to be done. I would appreciate any help to make my posh work with the remote SSH/docker terminal

**Comment by JanDeDobbeleer** [comment]
@ammaryasirnaich you need to configure the shell (and thus install oh-my-posh) on the target environment as well. there's no such thing as prompt forwarding, you're literally in another shell and runtime environment.

**Reply by ammaryasirnaich** [reply]
@JanDeDobbeleer, it makes sense, many thanks for the response :+1:
